.github/workflows/release-128tech.yaml

```diff
@@ -0,0 +1,30 @@
+name: Release pipeline for 128tech RPMs
+
+on:
+  release:
+    tags: [128tech-v*]
+
+jobs:
+  build:
+    name: Build RPMs
+    runs-on: ubuntu-latest
+    env:
+      VERSION_TAG: ${{ github.event.release.tag_name }}
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: Extract the version from the tag name
+        # tag expected to be of the form 128tech-v<version-number>
+        run: echo "::set-env name=VERSION::${VERSION_TAG:9}"
+
+      - name: Build some RPMs
+        uses: ./.github/actions
```

Comment:
Will this run all actions that we have configured?

Reply:
Yeah, it's a little strange. There can only be one action per `action.yaml` file and one `action.yaml` file in a given directory. So, pointing to a directory for actions to use is actually just saying to run the single action you'll find there. The directory structure could be updated to `.github/actions/rpm-build/action.yaml` to allow for the addition of other actions, but I'll leave that for whenever/if ever it becomes necessary.